ban/http/schema.py

```diff
@@ -24,6 +24,15 @@
         'url': 'https://adresse.data.gouv.fr/api-gestion/',
     },
     'paths': {},
+    'tags': [
+        {'name': 'Municipality'},
+        {'name': 'Postcode'},
+        {'name': 'Group'},
+        {'name': 'Housenumber'},
+        {'name': 'Position'},
+        {'name': 'Diff'},
+        {'name': 'User'},
+    ],
```

Comment:
Using Tags seems to me like a workaround to some other issue which I don't know.
So the issue we are trying to address should be expressed, and explained why tags is the right solution to it.